cmd/apiregister-gen/generators/parser.go

```diff
@@ -324,6 +331,25 @@ func (b *APIsBuilder) GetNameAndImport(tags SubresourceTags) (string, string) {
 	return strings.Join([]string{pkg, tags.RequestKind}, "."), importPackage
 }
 
+// ResourceTags contains the tags present in a "+resource=" comment
+type ResourceTags struct {
+	Resource string
+	REST     string
+}
+
+// ParseResourceTag parses the tags in a "+resource=" comment into a ResourceTags struct
+func ParseResourceTag(tag string) ResourceTags {
+	args := strings.Split(tag, ",")
+	rest := ""
+	if len(args) == 2 {
```

Comment:
Not a blocker for this PR, but I am thinking we will want to change the format of the tag to be more user friendly.

e.g.

```go
// +resource=name:foo,rest:FooRESTImpl
```

And the same for subsourcee

```go
// +subresource=name:scale,request:Scale,rest:ScaleUniversityREST
```